packages/bpk-component-datepicker/src/BpkDatepicker.js

```diff
@@ -40,6 +40,17 @@ class BpkDatepicker extends Component {
     };
   }
 
+  componentDidUpdate(prevProps) {
+    const { isOpen } = this.props;
+
+    if (prevProps.isOpen !== isOpen) {
+      // eslint-disable-next-line react/no-did-update-set-state
+      this.setState({
+        isOpen,
+      });
```

Comment:
Would it make sense for these to call the existing `onOpen` and `onClose` methods? That way if those methods are responsible for anything else (such as calling `props.onClose`) then we don't need to duplicate the logic?

Reply:
That makes sense, much cleaner this way. Fixed now